src/test/bundle_test.ts

```diff
@@ -35,7 +37,7 @@ class FileTransform extends Transform {
     this.transform = transform;
   }
   _transform(f: File, _encoding: string, cb: () => void) {
-    this.transform(this, f);
+    this.transform(this, f.clone());
```

Comment:
This just addressed a bug in the tests themselves.  The old tests passed either way, but they didn't prove what they needed to without the clone.